### DISCUSSION TITLE
Does this disrupt internet/TV if I don't have KPN yet with KPN configuration.

### DISCUSSION THREAD

**Discussion by Sculas** [fabianishere/udm-iptv#65] [Q&A] [answered]
Hi there!

I currently have Ziggo as my ISP and I'll be getting KPN in a week or so. If I install and configure this tool with the KPN configuration right now (while I still have Ziggo), will this disrupt internet or TV? Or will this only "have effect" when I've switched over to KPN?

**Comment by fabianishere** [comment, marked as answer]
The suggested WAN settings won’t work with your current Ziggo setup, so you’ll have to configure those once your KPN subscription becomes active.

The helper script won’t affect your service.

**Reply by Sculas** [reply]
I already figured that the KPN configuration doesn't work with Ziggo, but that's not a problem right now. I asked this since I wanted to know if I can configure this before my subscription becomes active so I don't need to do this after, so TV keeps working right after.

**Reply by fabianishere** [reply]
Yes, that should be possible :)

**Reply by Sculas** [reply]
~~Can I configure this script right now, or should I wait until my KPN subscription becomes active and then configure it?~~

Answered ^

**Reply by Sculas** [reply]
Thank you!

**Reply by fabianishere** [reply]
For KPN, the default settings should work in most cases.
